### PR TITLE
New package: Trimble.SketchUp.Pro.2023 version 23.1.340

### LINKED ISSUE #146641: Trimble.Sketchup.Pro.2023
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

Trimble.Sketchup.Pro.2022

### Package Version

23.1.340

### Please describe the changes you would like to see

_No response_

### PR DESCRIPTION
**Pull request by Dragon1573**
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] There's a linked issue. Resolve #146641

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/146649)

### CODE REVIEW

**Comment by sanderwi-ilionx** [comment]
https://download.sketchup.com/SketchUpStudio-2023-1-340-117.exe

File: manifests/t/Trimble/SketchUp/Pro/2023/23.1.340/Trimble.SketchUp.Pro.2023.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+PackageLocale: en-US
+Publisher: Trimble, Inc.
+PublisherUrl: https://sketchup.com/
+PublisherSupportUrl: https://help.sketchup.com/en
+PrivacyUrl: https://www.sketchup.com/legal/privacy-notice
+Author: Trimble, Inc.
+PackageName: SketchUp 2023
+PackageUrl: https://help.sketchup.com/en/downloading-sketchup
+License: Proprietary
+LicenseUrl: https://www.sketchup.com/license/b/sketchup-pro
```
Comment (stephengillie) [inline manifests/t/Trimble/SketchUp/Pro/2023/23.1.340/Trimble.SketchUp.Pro.2023.locale.en-US.yaml:15]:
```suggestion
PublisherUrl: https://sketchup.com/
# PublisherSupportUrl: 
# PrivacyUrl: 
Author: Trimble, Inc.
PackageName: SketchUp 2023
# PackageUrl: 
License: Proprietary
# LicenseUrl: 
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @Dragon1573,

This is probably due to an IP block of data centers, to prevent automated connections. Different companies might block with different goals - some might want to reduce network spend, while others might be trying to increase their security posture.

**Review by Dragon1573 — COMMENTED** [review]
Several URLs prevent data centers from visiting the site. Just remove them from the manifest to avoid complaint by Azure Pipelines.

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
image

- https://github.com/microsoft/winget-pkgs/issues/107639

**Pull request merged**